src/views/CustomerStory.vue

```diff
@@ -0,0 +1,133 @@
+        :class="{ 'is-active': showNav }"
+        @click="toggleNav()"
+      >
+        <div class="navbar-end">
+          <router-link to="/" class="navbar-item" href="#">Home</router-link>
+          <LocaleChanger :langs="{ en: 'English', de: 'Deutsch' }" />
+        </div>
+      </div>
+    </nav>
+
+    <br />
+    <br />
+    <section class="section">
+      <div class="columns is-centered columns-section">
+        <div class="column is-half">
+          <h2 class="title is-2 is-spaced">
+            {{ customer.title }}
+          </h2>
+          <figure class="image customer-story-logo">
+            <img :src="logo" :alt="customer.logo.alt" />
+          </figure>
+          <br />
+          <p>
+            <i>"{{ customer.quote.text }}"</i>
+          </p>
+          <br />
+          <p style="float: right">
+            {{ customer.quote.stakeholder.name }},
+            {{ customer.quote.stakeholder.role }}
+          </p>
+          <br />
+          <br />
+          <h3 class="title is-3">{{ $t("customer_story.customer") }}</h3>
+          <p>
+            <span v-html="customer.customer"></span>
+          </p>
+          <br />
+          <h3 class="title is-3">{{ $t("customer_story.background") }}</h3>
+          <p>
+            <span v-html="customer.background"></span>
+          </p>
+          <br />
+          <h3 class="title is-3">{{ $t("customer_story.goal") }}</h3>
+          <p>
+            <span v-html="customer.goal"></span>
+          </p>
+          <br />
+          <h3 class="title is-3">{{ $t("customer_story.contribution") }}</h3>
+          <p>
+            <span v-html="customer.contribution.intro"></span>
+          </p>
+          <br />
+          <div v-for="topic in customer.contribution.topics" :key="topic.title">
+            <h4 class="title is-4">{{ topic.title }}</h4>
+            <p>
+              <span v-html="topic.text"></span>
+            </p>
+            <br />
+          </div>
+          <h3 class="title is-3">{{ $t("customer_story.technologies") }}</h3>
+          <ul style="list-style-type: circle; padding-left: 30pt">
+            <li v-for="technology in customer.technologies" :key="technology">
+              {{ technology }}
+            </li>
+          </ul>
+          <br />
+        </div>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script>
+import Customers from "../customer/customer.js";
+import LocaleChanger from "../components/LocaleChanger.vue";
+
+export default {
+  components: {
+    LocaleChanger,
+  },
+  data() {
+    return {
+      showNav: false,
+    };
+  },
+  computed: {
+    customer: function () {
+      return Customers[this.$route.params.name][this.$i18n.locale];
+    },
+    logo: function () {
+      return require("../assets/" +
+        Customers[this.$route.params.name][this.$i18n.locale].logo.file);
+    },
+  },
+  methods: {
+    toggleNav() {
+      this.showNav = !this.showNav;
+    },
+  },
+};
+</script>
```

Comment:
Please add a blank line between `</script>` and `<style>`.

Reply:
Done.